src/containers/pointsContainer.js

```diff
@@ -135,6 +169,26 @@ class PointsContainer extends Component {
     this.setState({ isClaiming: false });
   };
 
+  _promote = async (duration, permlink, author, user) => {
+    const { currentAccount, pinCode, dispatch, intl, navigation } = this.props;
+    this.setState({ isLoading: true });
+
+    await promote(user || currentAccount, pinCode, duration, permlink, author)
+      .then(() => {
+        this.setState({ isLoading: false });
+        dispatch(toastNotification(intl.formatMessage({ id: 'alert.successful' })));
+        navigation.goBack();
+      })
+      .catch(error => {
+        Alert.alert(
+          `Fetching data from server failed, please try again or notify us at info@esteem.app \n${error.message.substr(
+            0,
+            20,
+          )}`,
```

Comment:
can you get error message from language file

Reply:
that is a different process its need another effort